x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg-utils
```

Sorry, I missed this earlier:
```suggestion
inherit meson xdg
```